src/pages/Home.tsx

```diff
@@ -1,6 +1,30 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Link } from 'react-router-dom';
 
+function toHexString (byteArray: Uint8Array): string {
+  return Array.from(byteArray, (byte: number) => {
+    return ('0' + (byte & 0xff).toString(16)).slice(-2);
+  }).join('');
+}
+
+const LogWindow = () => {
+  const [message, setMessage] = useState('');
+
+  useEffect(() => {
+    // @ts-ignore
+    import('sesamoid').then(({ KeyPair, setWasmPanicHook }) => {
```

Comment:
Why are you not importing this at the top? @adzialocha

Reply:
This is not a JavaScript `import` but a special async wasm import. Putting this to the top would not reflect the async nature of this eg. the components using the wasm methods would not know if the methods already got imported or not.

I also written it like this due to the experimental nature of this PR, if we would need to write it properly I would suggest writing a hook `useSesamoid` for this, something like:

```js
useSesamoid(({ KeyPair }) => {
  const keyPair = new KeyPair();
  // .. do more with wasm methods
});
```

The hook itself would make sure to only import the wasm once (singleton) when being used multiple times across the app.

Reply:
ok thank you!